### PR TITLE
Feature/basis 50

### PR DESCRIPTION
**Pull request by dvarabyouragnarok**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by mart-e** [comment]
Hello,

Thanks for your contribution but what are you wanting to change exactly.
If this is something you want to be merged, please document your changes.

File: addons/point_of_sale/i18n/uk.po [hunk 1]
```diff
@@ -535,6 +535,13 @@ msgstr "Додаткова необхідна інформація корист
 msgid "Address"
 msgstr "Адреса"
 
+#. module: point_of_sale
+#. odoo-javascript
+#: code:addons/point_of_sale/static/src/xml/Screens/PartnerListScreen/PartnerListScreen.xml:0
+#, python-format
+msgid "Student Card"
+msgstr "Студентський квиток"
```
[the remark this answers is not in the document]
Comment (dvarabyouragnarok) [reply addons/point_of_sale/i18n/uk.po]: Done

[the remark this answers is not in the document]
Comment (mart-e) [reply addons/point_of_sale/i18n/uk.po]: and translations should not be done on github but [on transifex](https://app.transifex.com/odoo/)

[the remark this answers is not in the document]
Comment (dvarabyouragnarok) [reply addons/point_of_sale/i18n/uk.po]: Hi Martin, sorry. This PR is wrong. It's not meant for this repo. Will cancel it. Thank you!

**Pull request closed without merging**